CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.25)
 
+SET(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake")
```

Comment:
setting the toolchain file in CMakeLists.txt is quite uncomfortable :(

Reply:
From what I read, it's either this, or everyone *has* to use some `cmake --preset="preset with this toolchain"` themselves on the command line